ui-backend/catalog-ui-search/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -253,6 +253,12 @@ Implementation details
         <argument ref="catalogFramework"/>
     </bean>
 
+    <reference-list id="revertProperties" interface="org.codice.ddf.catalog.ui.metacard.internal.PropertySupplier" filter="(type=revert)"/>
```

Comment:
I'm using a filter to establish a convention on how downstream projects can supply properties for other aspects of ddf-ui.